js/client.js

```diff
@@ -26,9 +26,13 @@
 		}
 	});
 	$(window).on('dragover', function (e) {
+		if (/^text/.test(e.target.type)) // Ignore text fields
```

Comment:
We also require if statements to be either:

    if (A) B;

or

    if (A) {
        B;
    }